packages/components/Checkbox/src/CheckboxTokens.ts

```diff
@@ -30,7 +30,7 @@ export const defaultCheckboxTokens: TokenSettings<CheckboxTokens, Theme> = (t: T
       checkboxBorderWidth: globalTokens.stroke.width.thin,
       checkboxBorderRadius: globalTokens.corner.radius.small,
       checkboxSize: 20,
-      checkmarkSize: 10,
+      checkmarkSize: 16,
```

Comment:
Can you show a screenshot of the large checkbox, too?

Reply:
Updated the PR description with a screenshot.

Reply:
Thanks!